packages/x-scheduler/src/joy/event-calendar/EventCalendar.css

```diff
@@ -37,3 +37,33 @@
   justify-content: center;
   color: var(--gray-6);
 }
+
+.EventCalendarResourceLegend {
```

Comment:
I will let the implementation of the full design done by Nora for #17842 :+1: 
I did a very simple version to improve readability.